components/login-form.html

```diff
@@ -0,0 +1,41 @@
+<div class="flex justify-center items-center h-screen bg-gray-200 px-6">
```

Comment:
That will allow it to be at full width in mobile.
```suggestion
<div class="flex justify-center items-center h-screen bg-gray-200 sm:px-6">
```